src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -308,8 +319,56 @@ export class BrowsingContextImpl {
           // Default execution context is set with key `null`.
           this.#sandboxToExecutionContextIdMap.set(null, params.context.id);
         }
+        if (params.context.name !== undefined) {
+          this.#sandboxToExecutionContextIdMap.set(
+            params.context.name,
+            params.context.id
```

Comment:
```suggestion
            params.context.id,
```

Reply:
Same. Prettier should be updated instead, but it's out of scope.